samza-core/src/test/java/org/apache/samza/serializers/model/TestSamzaObjectMapper.java

```diff
@@ -279,6 +279,28 @@ public void testDeserializeSystemStreamPartition() throws IOException {
     assertEquals(ssp, deserSSP);
   }
 
+  @Test
+  public void testSerdeSystemStreamPartitionKey() throws IOException {
```

Comment:
Could we add a test, that checks if new SamzaObjectMapper is able to deserialize serialized data written by old SamzaObjectMapper, 
and 
vice versa (Old SamzaObjectMapper is able to "handle" data written by new SamzaObjectMapper) -- for this one if no test is feasible, just a manual verification for now?